CONTRIBUTING.md

```diff
@@ -26,10 +26,42 @@ If you've found a bug, or have a feature idea that you you'd like to see, follow
 
 ### Contributing Code
 
-If you are new to the repo, you might want to pay close attention to these tags, as they are typically a great way to get started: Good First Bug, Bug, Chore, and Polish. If you see a bug that is not yet assigned to anyone, start a conversation with an engineer in the ticket itself, expressing your interest in taking the bug. If you take the bug, someone will set the ticket to Assigned to Contributor, so we can be proactive about helping you succeed in fixing the bug.
+If you are new to the repo, you might want to pay close attention to these tags, as they are typically a great way to get started:
+
+- Good First Bug
+- Bug
+- Chore
+- Polish
+
+If you see a bug that is not yet assigned to anyone, start a conversation with an engineer in the ticket itself, expressing your interest in taking the bug. If you take the bug, someone will set the ticket to Assigned to Contributor, so we can be proactive about helping you succeed in fixing the bug.
 
 When you have some code written, you can open up a Pull Request, get your code reviewed, and see your code merged into the codebase.
 
+### Commit format
+
+We use standard changelog and enforce [conventional commits](https://www.conventionalcommits.org/).
+
+Commits should be formated as `type(scope): message`
```

Comment:
I think some examples would be useful here.

Have we documented anywhere what _scope_ should be? My understanding is that it's a freeform string, that's set to the name of the component that's being changed.